automat/automat.ino

```diff
@@ -441,6 +442,25 @@ void receiveI2CEvent(int len)
         }
       }
       break;
+    case I2C_MIDI_SET:              // received MIDI Event set containing up to 3 Bytes (=1)
+      if (len = 4) {
+        uint8_t byte1 = Wire.read();
+        uint8_t byte2 = Wire.read();
+        uint8_t byte3 = Wire.read();
+
+        uint8_t header = byte1 & 0xF0;
+        switch (header) {
+          case 0x80:  // Note-off
+              handleNoteOff(1 + (byte1 & 0x0F), byte2, byte3);
+              break;
+          case 0x90:  // Note-on
+              handleNoteOn(1 + (byte1 & 0x0F), byte2, byte3);
```

Comment:
MIDI Note-on messages with a velocity of 0 should be treated as a note-off event.

Reply:
So if byte3 == 0 for the 0x90 case, you should call handleNoteOff(...)

Reply:
thanks lefty71. of course. I'm already filtering those zero velocity events at the sender side :)

Reply:
added a new pull request with filter also included on receiver side :)